src/main/java/com/revature/controllers/PersonController.java

```diff
@@ -0,0 +1,91 @@
+package com.revature.controllers;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.http.converter.HttpMessageNotWritableException;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.revature.domain.Person;
+import com.revature.services.PersonLogic;
+
+@RestController
+@RequestMapping(value = "/api/v1/")
+public class PersonController {
+	
+	@Autowired
+	private PersonLogic personLogic;
+	
+//	@RequestMapping(method = RequestMethod.GET, value = "person")
+//	public ResponseEntity<List<Person>> getAllPerson() {
+//		
+//		return ResponseEntity.ok(personLogic.getAllPersons());
+//	}
+	
+	@RequestMapping(method = RequestMethod.GET, value = "person")
+	public ResponseEntity<List<Person>> getPerson(@RequestParam(defaultValue="", required=false) String firstname,
+			@RequestParam(defaultValue="", required=false) String lastname, 
+			@RequestParam(defaultValue="0", required=false) Integer id,
+			@RequestParam(defaultValue="0", required=false) Integer role){
+		
+		 if((id==0)&&(role==0)){
+			 			 
+			 if(!firstname.equals("")){
+				 return ResponseEntity.ok(personLogic.getPersonByFirstName(firstname));
+				 
+			 } else if (!lastname.equals("")){
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Move the "" string literal on the left side of this string comparison. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.revaturelabs.com/coding_rules#rule_key=squid%3AS1132)